src/tests/unit/config.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { configureStore, combineReducers } from "@reduxjs/toolkit";
```

Comment:
Good stuff adding tests.

I wonder if there is a reason why this does not use in file testing?

Reply:
Becase "@testing-library/react-hooks" is a dev dependency and the config.ts file is a production file